addons/hr/static/src/scss/hr.scss

```diff
@@ -8,54 +8,28 @@
         display: flex;
         flex-direction: row;
         align-items: center;
-        margin-bottom: 5px;
+        margin-bottom: map-get($spacers, 1);
```

Comment:
commit f0b78edb68f68ceebc44055181ed08fae75f3457
Is it possible to add these classes directly on the `hr_employee_chat` widget instead?

Reply:
Funny enough, when I first worked on the revamp, `o_employee_chat_btn` could be called within a form or a kanban view, so it was a bit tricky to apply conditonal classes, but it looks like the button is no more displayed within kanban views so we might be able to fix it